ios/Framework/Core/DDL/FormScreenlet/DDLFormScreenlet.swift

```diff
@@ -356,10 +356,10 @@ import UIKit
 
 	//MARK: Private methods
 
-	private func waitForInProgressUpload() -> Bool {
+    private func waitForInProgressUpload(interactor: Interactor) -> Bool {
```

Comment:
change your Xcode settings to use tabs, review everything and add a new commit with that